accounts/models.py

```diff
@@ -1,18 +1,26 @@
-from django.db import models
+﻿from django.db import models
 from django.contrib.auth.models import User
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 
+
 class Profile(models.Model):
-    user = models.OneToOneField(User, on_delete=models.CASCADE)
-    email = models.EmailField(max_length=150)
-    signup_confirmation = models.BooleanField(default=False)
+    user = models.OneToOneField(User, on_delete=models.CASCADE, related_name='profile')
+    college_major = models.CharField(max_length=20, blank=True)
+    Class = models.CharField(max_length=20, blank=True)
```

Comment:
normally classes are named with initial capital in python. Do not name normal variables with initial capital, it is not a good practice.